package.json

```diff
@@ -4,34 +4,30 @@
   "description": "A minimalist and highly composable UI toolkit built on top of React and styled-components",
   "license": "MIT",
   "repository": "reakit/reakit",
-  "main": "dist/cjs/reakit.js",
-  "module": "dist/es/reakit.js",
-  "jsnext:main": "dist/es/reakit.js",
-  "unpkg": "dist/umd/reakit.min.js",
+  "main": "lib/reakit.js",
+  "module": "es/reakit.js",
+  "browser": "dist/reakit.min.js",
```

Comment:
⚠️  Do not put umd files in browser field, this is a common misconception. 

webpack (& other tools) will prioritize browser over module field and we want it to pick ESM formatted files. browser field can be used for a platform specific code, take a look on how I use it [here](https://github.com/andreypopp/react-textarea-autosize/blob/01717390e4bf59aaa9ecdadb0a6a03f4f761fc67/package.json#L10-L13). In that case it's just a really minor micro optimization, but I'm dropping runtime browser detection and some SSR guards from "regular" files to provide more "optimized" (byte-wise) browser files.